musicbot/bot.py

```diff
@@ -3096,12 +3098,13 @@ async def cmd_autoplaylist(
         guild: discord.Guild,
         author: discord.Member,
         _player: Optional[MusicPlayer],
+        player: MusicPlayer,
         option: str,
         opt_url: str = "",
     ) -> CommandResponse:
         """
         Usage:
-            {command_prefix}autoplaylist [ + | - | add | remove] [url]
+            {command_prefix}autoplaylist [ + | +all | - | add | add_all | remove] [url]
```

Comment:
May be worth adding a second line since the `+all|add_all` sub-commands do not take the `[url]` command parameter